cli/src/merge.js

```diff
@@ -376,12 +376,16 @@ class Merge {
     delete oldMetadata.errors
     const newMetadata = clone(oldMetadata)
     markSide(side, newMetadata, oldMetadata)
+    newMetadata._id = doc._id
+    newMetadata._rev = doc._rev
+    newMetadata.path = oldMetadata.path
```

Comment:
Why do you set the `path` of `newMetadata` from `oldMetadata`? `newMetadata` was cloned from `oldMetadata`, so I'd expected them to be the same